src/vs/editor/common/controller/cursor.ts

```diff
@@ -1126,6 +1128,10 @@ export class Cursor extends EventEmitter {
 		return this._invokeForAll(ctx, (cursorIndex: number, oneCursor: OneCursor, oneCtx: IOneCursorOperationContext) => OneCursorOp.moveTo(oneCursor, inSelectionMode, ctx.eventData.position, ctx.eventData.viewPosition, ctx.eventSource, oneCtx));
 	}
 
+	private _move(inSelectionMode:boolean, ctx: IMultipleCursorOperationContext): boolean {
```

Comment:
`inSelectionMode` should be added as a boolean argument accepted by the command. Not sure what name would be best.